src/activities/morse_code/MorseCode.qml

```diff
@@ -370,12 +376,81 @@ ActivityBase {
 
         }
 
+        GCText {
+            id: morse_map
+            wrapMode: TextEdit.WordWrap
+            fontSize: mediumSize
+            horizontalAlignment: Text.AlignHCenter
+            anchors.horizontalCenter: parent.horizontalCenter
+            anchors.verticalCenter: parent.verticalCenter
+            width: parent.width * 0.8
+            height: parent.height * 0.8
+            visible: false
+            text: qsTr("\nA = .-\tB = -...\tC = -.-.\tD = -..\tE = .")
```

Comment:
I would not use qsTr there, the code is universal (if I'm not wrong), we won't translate it.